index.js

```diff
@@ -0,0 +1,53 @@
+import UI from './modules/ui.js';
+import Book from './modules/book.js';
+import formatDate from './modules/date.js';
+
+// Event listeners
+document.getElementById('bookForm').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.getElementById('titleInput').value;
+  const author = document.getElementById('authorInput').value;
+  const book = new Book(title, author);
+  if (title === '' || author === '') {
+    UI.showAlert('Please fill in all fields', 'error');
+  } else {
+    UI.addBookToList(book);
+    UI.clearInputFields();
+    UI.saveToLocalStorage(book);
+    UI.showAlert('Book added', 'success');
+  }
+});
+document.getElementById('bookList').addEventListener('click', (e) => {
+  UI.deleteBookFromList(e.target);
+});
```

Comment:
- [ ] When deleting a book from a list, the `localstorage` is not updated. Please update the `localstorage` when a book gets deleted so that a refresh of the page doesn't show the deleted book 👍

Reply:
@codepantha I have resolved this issue.